index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require_once "vendor/autoload.php";
+
+$db = new GRUB\Db();
+$ingredientHydrator = new GRUB\IngredientHydrator($db->getDB());
```

Comment:
Now that the getDB method is static you don't need to instantiate a new Db object, you can just use GRUB\Db::getDB() directly in the params for the hydrator